lib/turn.rb

```diff
@@ -0,0 +1,26 @@
+
+
+class Turn
+  attr_reader :guess,:card
+
+  def initialize(guess,card)
+    @guess = guess
+    @card = card
+  end
+
+  def correct?
+    if @guess == card.answer
+    true
+      else
+        false
+      end
```

Comment:
this method could be shortened to just one line: 

```
@guess == card.answer
```

becuase this will automatically return true or false. 

Also, I mentioned this before, but just to reiterate, I'd call to `@card` instead of `card`. Both will work, but you might as well call directly to the variable instead of to the method that returns that variable.